src/com/kryvapust/Infrastructure/Aviaries.java

```diff
@@ -0,0 +1,68 @@
+package com.kryvapust.Infrastructure;
+
+import com.kryvapust.Dogs.Dog;
+import com.kryvapust.Dogs.TypesOfDog;
+import com.kryvapust.Employees.Employee;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class Aviaries {
+
+    private Collection<Dog> allDogs;
+    private Collection<Dog> temporaryPlaceForDogs = new HashSet<Dog>();
+    private Collection<Dog> dogsForWork = new HashSet<Dog>();
+    private Collection<Dog> dogsForTraining = new HashSet<Dog>();
+    private Collection<Dog> restOfDogs = new HashSet<Dog>();
+
+    Collection<Dog> getAllDogs() {
+        return allDogs;
+    }
+
+    void setAllDogs(Collection<Dog> allDogs) {
+        this.allDogs = allDogs;
+    }
+
+    Collection<Dog> getTemporaryPlaceForDogs() {
+        return temporaryPlaceForDogs;
+    }
+
+    Collection<Dog> getDogsForWork() {
+        return dogsForWork;
+    }
+
+    Collection<Dog> getDogsForTraining() {
+        return dogsForTraining;
+    }
+
+    Collection<Dog> getRestOfDogs() {
+        return restOfDogs;
+    }
+
+
```

Comment:
I can use this method only for staff, that comes into aviaries and do operation with all dogs, but if I add new input staff's  functions this method will be useless.